python/orca/dev/example/run-example-test-ray.sh

```diff
@@ -63,6 +66,7 @@ start=$(date "+%s")
 python ${BIGDL_ROOT}/python/orca/example/ray_on_spark/parameter_server/sync_parameter_server.py --iterations 10
 now=$(date "+%s")
 time7=$((now-start))
+fi
```

Comment:
will it throw error when time6 and time7 are not defined?

Reply:
no, just output nothing